sdcm/tester.py

```diff
@@ -2301,7 +2302,7 @@ def create_table(self, name, key_type="varchar",  # pylint: disable=too-many-arg
             prefix = ' AND compaction={'
             postfix = '}'
             compaction_clause = " 'class': '%s'" % compaction
-            if sstable_size:
+            if sstable_size and compaction == CompactionStrategy.LEVELED.value:
```

Comment:
ICS also supports `sstable_size_in_mb` so I suppose it should be enabled for `CompactionStrategy.INCREMENTAL` too.

Reply:
right, fixed